pkg/pillar/hypervisor/kvm.go

```diff
@@ -875,7 +875,7 @@ func waitForQmp(domainName string, available bool) error {
 			// Give up
 			logrus.Warnf("waitForQmp for %s %t: giving up", domainName, available)
 			if available {
-				return logError("Qmp not found: error %v", err)
+				return logError("Giving up waiting to start VM: %s, error in QEMU Monitor Protocol (Qmp): %v", domainName, err)
```

Comment:
The `waitForQmp` function is not only used during VM start but also in case of VM stopping and it could potentially be used for other cases. Therefore, I suggest placing messages related to specific use cases in the function callers rather than in the function itself. In this case, I think https://github.com/lf-edge/eve/blob/4290b8e51d5e3e6b3336404f1e9ee295181db416/pkg/pillar/hypervisor/kvm.go#L898 is more suitable.

Reply:
You are right, but this is if available is false. So inside this if, the log is accurate.

Reply:
Yep, but we can still use the function with `available` set to true for any other purpose, for example, for some kind of watchdog functionality. Only the caller can know what function is used for.

Reply:
You have a point. What about `return logError("Giving up waiting to connect with VM: %s, error in QEMU Monitor Protocol (Qmp): %v", domainName, err)`

Reply:
Yeah, it's more suitable for this place. But I would also add a more explicit error message to the caller, as now it just prints info about QMP and says nothing about VM start failure.

Reply:
@jsfakian , I agree with @OhmSpectator . Also, `error in QEMU Monitor Protocol (Qmp)` sounds like an error on the protocol itself, but wouldn't be the case if the socket file was not found.... maybe something like the following would be more clear: 

`Giving up waiting to connect to QEMU Monitor Protocol socket ... from VM ....`